onTruckPlugin/src/main/java/com/github/ontruck/OnTruck.java

```diff
@@ -1,32 +1,51 @@
 package com.github.ontruck;
 
+import com.github.ontruck.filters.DMSFilter;
+import com.github.ontruck.filters.FilterManager;
+import com.github.ontruck.filters.ManualFilter;
+
 import java.io.IOException;
 import java.net.SocketException;
 
-public class OnTruck {
+public class OnTruck implements Runnable {
```

Comment:
Some comments explaining how filters, filtermanager and the driver interface works in relation to eachother would be nice